crafting_tweaks/files/quality_of_life/double_slabs/recipes/double_slabs/polished_diorite_slab.recipe.json

```diff
@@ -19,8 +19,7 @@
 			}
 		],
 		"result": {
-			"item": "minecraft:stone_block_slab3",
-			"data": 5,
+			"item": "minecraft:polished_andesite_slab",
```

Comment:
Should be `"minecraft:polished_diorite_slab"` not `"minecraft:polished_andesite_slab"`